hivemq-edge/src/main/java/com/hivemq/api/model/adapters/ProtocolAdapter.java

```diff
@@ -66,6 +66,10 @@ public class ProtocolAdapter {
     @Schema(description = "Is the adapter installed?")
     private final @NotNull Boolean installed;
 
+    @JsonProperty("supportsDiscovery")
```

Comment:
Do we want to provide a list of capabilities of an adapter, e.g., writeable, readable, discoverable.

This would lead to an array of string rather than a boolean which would be things more extendable.

Reply:
Very good point. Let's use capability constants. I will add.